src/views/SalesInfo/index.tsx

```diff
@@ -1,53 +1,129 @@
+              <P2 color={colors.white} center={true}>
+                {' '}
+                Welcome to the wPOKT distribution event. This distribution is the first opportunity to acquire wPOKT
+                that will enable holders to participate in wPOKT. The distribution will occur on Balancer using a
+                Liquidity Bootstrapping Pool. Before participating in wPOKT, we highly encourage users to read all of
+                the wPOKT documentation and information located in the links below. Welcome aboard and happy farming.
+              </P2>
+              <Spacer size={'md'} />
+              <H2 color={colors.white} center={true}>
+                WPOKT: 0,00006373 USDC/ DAI
+              </H2>
+            </StyledInfoCard>
+          </StyledContentContainer>
+          <Spacer size={'md'} />
+          <Flex
+            css={`
+              flex-wrap: wrap;
+            `}
+            align={'flex-start'}
+            justify={'space-between'}
+          >
+            <div>
+              <SalesInfoStatCard statContent={'2:00:00:00'} statTitle={'Sale Ends'} />
+              <Spacer size={'xs'} />
+            </div>
+            <div>
+              <SalesInfoStatCard statContent={'1.98 USDC'} statTitle={'Latest Price'} />
+              <Spacer size={'xs'} />
+            </div>
+            <div>
+              <SalesInfoStatCard statContent={'4,080'} statTitle={'Farmers'} />
+              <Spacer size={'md'} />
+            </div>
+            <div>
+              <SalesInfoStatCard statContent={'9,332,236 USDC'} statTitle={'USDC Traded'} />
+              <Spacer size={'md'} />
+            </div>
+          </Flex>
           <StyledContentContainer>
             <StyledContentTextContainer>
               <H1 color={colors.white}>wPOKT Liquidity bootstrapping Event</H1>
             </StyledContentTextContainer>
             <StyledChartContainer>
-              <BarChart width={600} height={300} data={data} margin={{ top: 20, right: 30, left: 20, bottom: 20 }}>
-                <XAxis xAxisId={0} dataKey="name" hide />
-                <XAxis xAxisId={1} dataKey="name" tickMargin={10} />
-                <YAxis domain={[0, 50000]} />
-                <Tooltip cursor={{ fill: 'transparent' }} />
-                <Bar barSize={10} xAxisId={0} dataKey="max" fill={colors.green} />
-                <Bar barSize={10} xAxisId={1} dataKey="min" fill={colors.white}>
-                  <ErrorBar dataKey="error" width={4} strokeWidth={2} stroke={colors.green} />
-                  <ErrorBar dataKey="errorNegative" width={4} strokeWidth={2} stroke={colors.red} />
-                </Bar>
-              </BarChart>
+              <LineChart
+                width={500}
+                height={300}
+                data={data}
+                margin={{
+                  top: 50,
+                  right: 30,
+                  left: 20,
+                  bottom: 30,
+                }}
+              >
```

Comment:
Will this look good on mobile?

Reply:
It sort of depends on how many decimals the price fetcher ends up displaying. But for safety, I've shrunk the graph text a bit more in order to lower the margin-left: d26d78f565224c4dfe602e4f371e6a155b8d0409